exec/java-exec/src/test/java/org/apache/drill/test/TestGracefulShutdown.java

```diff
@@ -237,7 +234,7 @@ public void testRestApi() throws Exception {
       }
 
       Assert.assertTrue(listener.isDone());
-      Assert.assertEquals(1,drillbitEndpoints.size());
+      Assert.assertEquals(2, drillbitEndpoints.size());
```

Comment:
Why did we have to change 1 to 2 here? Wasn't this test already passing?

Reply:
Now we are shutting down only one drillbit instead of two.

Reply:
K thx.